apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -74,6 +75,12 @@ type Job struct {
 
 	// Defines the restart policy for the CronJob, defaults to never
 	RestartPolicy v1.RestartPolicy `json:"restartPolicy,omitempty"`
+
+	// Defines the concurrency policy for the CronJob, defaults to Forbid
```

Comment:
Should we default to Forbid or Allow? I think the kube standard is Allow. 
https://v1-16.docs.kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy